projects/cdk/schematics/ng-update/v4/tests/schematic-migrate-package-names.spec.ts

```diff
@@ -0,0 +1,89 @@
+import {join} from 'node:path';
+
+import {HostTree} from '@angular-devkit/schematics';
+import {SchematicTestRunner, UnitTestTree} from '@angular-devkit/schematics/testing';
+import type {TuiSchema} from '@taiga-ui/cdk/schematics/ng-add/schema';
+import {
+    createProject,
+    createSourceFile,
+    resetActiveProject,
+    saveActiveProject,
+    setActiveProject,
+} from 'ng-morph';
+
+const collectionPath = join(__dirname, '../../../migration.json');
+
+const TS_FILE_BEFORE = `
+import {Component} from '@angular/core';
+import {PolymorpheusComponent} from '@tinkoff/ng-polymorpheus';
+import {shouldCall} from '@tinkoff/ng-event-plugins';
+import {NG_EVENT_PLUGINS} from '@tinkoff/ng-event-plugins';
+import {TUI_VERSION} from '@taiga-ui/cdk';
+`.trim();
+
+const TS_FILE_AFTER = `
+import {Component} from '@angular/core';
+import {PolymorpheusComponent} from '@taiga-ui/polymorpheus';
+import {shouldCall} from '@taiga-ui/event-plugins';
+import {NG_EVENT_PLUGINS} from '@taiga-ui/event-plugins';
+import {TUI_VERSION} from '@taiga-ui/cdk';
+`.trim();
+
+const PACKAGE_JSON_BEFORE = {
+    dependencies: {
+        '@angular/core': '~13.0.0',
+        '@taiga-ui/addon-commerce': '~3.42.0',
+        '@tinkoff/ng-polymorpheus': '1.2.3',
+        '@tinkoff/ng-event-plugins': '4.5.6',
+    },
+};
+
+const PACKAGE_JSON_AFTER = {
+    dependencies: {
+        '@angular/core': '~13.0.0',
+        '@taiga-ui/addon-commerce': '~3.42.0',
+        '@taiga-ui/event-plugins': '^4.0.1',
+        '@taiga-ui/polymorpheus': '^4.6.3',
```

Comment:
`cdkPackage.peerDependencies['@taiga-ui/polymorpheus']`

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/7947/commits/0fe148976a12234a0d9944f202a935b2d81a0372